### DISCUSSION TITLE
下载的mp3打不开

### DISCUSSION THREAD

**Discussion by Kanbe-Kotori** [1715173329/UnblockNeteaseMusic#329] [Q&A]
试了几首歌都这样，后台并没报错。然后播放似乎并没受影响。
使用的版本是v0.27.0-beta.9的src，命令行运行node app.js跑起来的

**Comment by pan93412** [comment]
使用 macOS 版 NeteaseMusic 2.3.6 (862) 內建下載功能 `Dua Lipa - New Rules (Initial Talk Remix)` 發現能在 QuickTime 正常播放。方便提供更多資訊嗎？

<img width="786" alt="NeteaseMusic 的「已下載單曲」" src="https://user-images.githubusercontent.com/28441561/141865545-83ee1511-1924-4e69-b1b1-bc409ce66fab.png">

<img width="462" alt="QuickTime" src="https://user-images.githubusercontent.com/28441561/141865787-4f92af9c-c078-4108-8592-16476f89a3ee.png">

**Comment by boxiqiexi** [comment]
去掉bilibili源再尝试下载
